exercises/reverse-string/canonical-data.json

```diff
@@ -1,8 +1,6 @@
 {
   "exercise": "reverse-string",
-  "comments": [
-        "If property based testing tools are available, a good property to test is reversing a string twice: reverse(reverse(string)) == string"
-      ],
+  "comments": ["If property based testing tools are available, a good property to test is reversing a string twice: reverse(reverse(string)) == string"],
```

Comment:
showing a place where a long comment was chosen to be the same line as the `[` and `]`